app/models/config.rb

```diff
@@ -32,9 +32,17 @@ class Config < ApplicationRecord
     voicemail: 12,
   }
 
+  # which fields are URLs
+  Config_URLs = %w[fax_service practical_support_guidance_url resources_url]
```

Comment:
This list is how the config model decides what is a URL.

Reply:
Looks right! Let's constantize it per convention and have it in all caps:

```suggestion
  CONFIG_URLS = %w[fax_service practical_support_guidance_url resources_url]
```